docs/astro/sql/operators/load_file.rst

```diff
@@ -0,0 +1,162 @@
+======================================
+load_file operator
+======================================
+
+When to Use load_file operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two use cases of the load_file operator.
+
+#. When we want a file to be loaded into a database table
```

Comment:
We should explain that we also support matching a group of files, and if there are limitations on our pattern matching at the moment (differences between local/gcs/s3).

Reply:
Done.